sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 2.0.0-beta.3 (Unreleased)
 
+- All of the credentials now allow specifying a default `correlationId` on the constructor options. If set, this default `correlationId` will be used if no `correlationId` is specified during the `getToken()` or `authenticate()` method calls.
```

Comment:
Is there an issue or service side docs that we can point to for anyone trying to understand how the correationId is used by the Identity service?

Reply:
I can mention that the correlationId is sent as the `client-request-id` parameter in the body of the outgoing requests. I'll add that.